avogadro/qtplugins/CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 find_package(Eigen3 REQUIRED)
 include_directories(SYSTEM ${EIGEN3_INCLUDE_DIR})
 
-find_package(Qt5 COMPONENTS Widgets Network Concurrent REQUIRED)
+find_package(Qt5Widgets REQUIRED)
```

Comment:
We don't use Qt5's OpenGL module anymore, and find Qt5 from the Qt5 config file to ensure consistency. This change should be edited out.